<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1030</width>
    <height>685</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>宇文通翻译</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QPushButton" name="openFolder">
        <property name="text">
         <string>打开文件夹</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGraphicsView" name="pageView"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="preButton">
          <property name="text">
           <string>上一张</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="NextButton">
          <property name="text">
           <string>下一张</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QGraphicsView" name="targetView"/>
          </item>
          <item>
           <widget class="QGraphicsView" name="sourceView"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5"/>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>输入翻译文字</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="transEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="saveButton">
          <property name="text">
           <string>保存图片</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
